src/AggregateFunctions/ThetaSketchData.h

```diff
@@ -66,6 +66,8 @@ class ThetaSketchData : private boost::noncopyable
             return 0;
     }
 
+    static void parallelizeMergePrepare(const std::vector<ThetaSketchData *> & /*places*/, ThreadPool & /*thread_pool*/) {}
```

Comment:
if I'm not missing smth all `parallelizeMergePrepare` could be one of:
* not defined/declared at all
* throw exception
* do meaningful work (empty implementation is not meaningful)

I have some prejudice towards empty methods, what do you think?

Reply:
Agree. 

- For these empty methods in other datasets, we could add the throw exception. 

- Or we may just delete these empty methods in other datasets. Call the `DataSet::parallelizeMergePrepare()` only when `is_parallelize_merge_prepare_needed` is true.

I would like to suggest the 2nd way.